src/github/github.api.mock.ts

```diff
@@ -22,3 +22,66 @@ export class GithubApiMock {
     });
   }
 }
+
+const openSourceRepository = {
+  licenseInfo: {
+    key: 'mit',
+  },
+  isFork: false,
+  isPrivate: false,
+};
+
+const notOpenSourceRepository1 = {
+  licenseInfo: null,
+  isFork: false,
+  isPrivate: false,
+};
+
+const notOpenSourceRepository2 = {
+  licenseInfo: {
+    key: 'mit',
+  },
+  isFork: true,
+  isPrivate: false,
+};
+
+export const graphqlResultMock = {
+  userInfo: {
+    id: 'githubusernameid',
+    login: 'githubusername',
+    name: 'Github User Name',
+    avatarUrl: 'https://githubusername.com',
+  },
+  totalContributions: {
+    contributionsCollection: {
+      repositoryContributions: {
+        nodes: [
+          { repository: notOpenSourceRepository1 },
+          { repository: openSourceRepository },
+        ],
+      },
+      commitContributionsByRepository: [
+        {
+          contributions: { totalCount: 44 },
+          repository: notOpenSourceRepository2,
+        },
+        {
+          contributions: { totalCount: 44 },
+          repository: openSourceRepository,
+        },
+      ],
+      issueContributionsByRepository: [
+        {
+          contributions: { totalCount: 5 },
+          repository: openSourceRepository,
+        },
+      ],
+      pullRequestContributionsByRepository: [
+        {
+          contributions: { totalCount: 6 },
+          repository: openSourceRepository,
+        },
+      ],
+    },
+  },
+};
```

Comment:
```suggestion
import response from './query-response.example.json';
```

I'd prefer the response to be a snapshot from an actual call, so it is easier to update (e.g. running the query against the API and copy-pasting the result). This ensures the data is coherent with the API response, rather than reducing repetition and potentially denaturing the response structure